ams/server/src/main/java/com/netease/arctic/server/persistence/mapper/OptimizingMapper.java

```diff
@@ -137,7 +138,7 @@ List<TableOptimizingProcess> selectSuccessOptimizingProcesses(
   void insertTaskRuntimes(@Param("taskRuntimes") List<TaskRuntime> taskRuntimes);
 
   @Select("SELECT process_id, task_id, retry_num, table_id, partition_data,  create_time, start_time, end_time," +
```

Comment:
the method updateTaskRuntime() should also add `optimizer_token` and `thread_id`